build.gradle

```diff
@@ -21,6 +22,13 @@ buildscript {
     }
 }
 
+plugins {
+    id("idea")
+    id("org.robolectric.gradle.SpotlessPlugin")
+}
+// nebula-aggregate-javadocs doesn't support plugins
+apply plugin: "nebula-aggregate-javadocs"
```

Comment:
The plugin even seem to be [no longer maintained](https://github.com/nebula-plugins/gradle-aggregate-javadocs-plugin). Maybe we should look for an alternative plugin that does the same thing.
Out of scope of this PR, but just asking.

Reply:
I've opened a ticker for this, so it doesn't get lost: https://github.com/robolectric/robolectric/issues/9196